alacritty.yml

```diff
@@ -3,16 +3,12 @@
 
 # Any items in the `env` entry below will be added as
 # environment variables. Some entries may override variables
-# set by alacritty it self.
+# set by alacritty itself.
 env:
   # TERM env customization.
   #
-  # If this property is not set, alacritty will set it to xterm-256color.
-  #
-  # Note that some xterm terminfo databases don't declare support for italics.
-  # You can verify this by checking for the presence of `smso` and `sitm` in
-  # `infocmp xterm-256color`.
-  TERM: xterm-256color
+  # If this property is not set, alacritty will set it to alacritty.
+  # TERM: alacritty
```

Comment:
The wording here is a bit awkward and the lacking space in between the description and the commented value makes it hard to see that it's actually a yml property.

Something like this might be better:
```
# TERM variable
#
# This value is used to set the `$TERM` environment variable for
# each instance of Alacritty. If it is not present, the default value
# 'alacritty' will be used instead.
#
# TERM: alacritty
```

Reply:
No argument there. Feel free to push a fix. Otherwise I'll do it later.